core/trino-main/src/test/java/io/trino/sql/query/QueryAssertions.java

```diff
@@ -451,15 +456,19 @@ public final QueryAssert isNotFullyPushedDown(PlanMatchPattern retainedSubplan)
                                 expectedPlan);
                     });
 
+            if (!skipVerifyingResultsWithPushdownDisabled) {
+                // Compare the results with pushdown disabled, so that explicit matches() call is not needed
+                verifyResultsWithPushdownDisabled();
+            }
             return this;
         }
 
-        private void verifyResultsWithPushdownDisabled()
+        private QueryAssert verifyResultsWithPushdownDisabled()
```

Comment:
Doesn't need to be fluent.

Reply:
Oh yes, it's private. Leftover from initial attempt.